src/share/poudriere/include/hybridset.sh

```diff
@@ -69,42 +69,41 @@ hybridset_list() {
 	(cd "${MASTER_DATADIR_ABS}" && find "hybridset" -type d -depth 2 | cut -d / -f 3 | sort -u)
 }
 
+hybridset_is_really_host() {
+	# XXX: Host ABI used as a proxy for whether we're building natively or
```

Comment:
NB: This probably should be looking at QEMU_EMULATING instead

Reply:
We should test if the host is FreeBSD instead.

Reply:
I thought you were building on CheriBSD/arm64 these days, so that would be insufficient.

Reply:
We also build CheriABI packages on FreeBSD/amd64.